testing/opencolorio/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Contributor: Mark Riedesel <mark@klowner.com>
+# Maintainer:
+pkgname=opencolorio
+pkgver=1.0.9
+pkgrel=0
+pkgdesc="A color management framework for visual effects and animation"
+url="http://opencolorio.org"
+arch="all"
+license="BSD"
+depends=""
+makedepends="cmake python2-dev"
+install=""
+subpackages="$pkgname-python $pkgname-dev"
+source="$pkgname-$pkgver.tar.gz::http://github.com/imageworks/OpenColorIO/archive/v$pkgver.tar.gz"
+builddir="$srcdir"/OpenColorIO-$pkgver
+
+build() {
+	cd "$builddir"
+	mkdir build && cd build
+	cmake -DCMAKE_INSTALL_PREFIX=/usr ..
```

Comment:
Please add `|| return 1` at the end of this line, to ensure that an error will halt the build (abuilds don’t run with `set -e` yet).